airbyte-integrations/connectors/source-surveymonkey/unit_tests/unit_test.py

```diff
@@ -0,0 +1,73 @@
+#
+# MIT License
+#
+# Copyright (c) 2020 Airbyte
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+import json
+
+import pendulum
+import vcr
+from airbyte_cdk.models import SyncMode
+from source_surveymonkey import SourceSurveymonkey
+
+
+def test_get_updated_state_unit():
+    source = SourceSurveymonkey()
+    config = json.load(open("secrets/config.json"))
+    streams = source.streams(config=config)
+    stream = [i for i in streams if i.__class__.__name__ == "Surveys"][0]
```

Comment:
this is a very convoluted way to create a `Surveys` stream. Why not do: 

```
stream = Surveys(<args>) 
```
? 

Also we should parametrize this test, and test for empty state